### PR TITLE
fix -- thing

### PR DESCRIPTION
**Pull request by BrianHicks**
primarily: fixes adding a `--` to direnv, which doesn't support that anymore

secondary: sets up a dev environment with nix-shell and stuff

🍐 with @michaelglass

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -1,28 +1,23 @@
...
+  version = "0.1.6";
+  src = gitignore.gitignoreSource ./.;
 
   # The crypto_hash crate needs the openssl-sys crate (directly or indirectly,
   # I don't know) which ultimately needs openssl proper, and pkg-config.
   buildInputs = [ openssl pkg-config ];
 
   # Don't run tests when building.
-  checkPhase = "";
+  doCheck = false;
```
Comment (michaelglass) [inline default.nix:16]: 😍

File: src/config.rs [hunk 2]
```diff
@@ -134,7 +134,7 @@ impl Config {
 
     pub fn command_to_allow_direnv(&self) -> Command {
         let mut command = self.command_direnv();
-        command.arg("allow").arg("--").arg(&self.build_dir);
+        command.arg("allow").arg(&self.build_dir);
```
Comment (michaelglass) [inline src/config.rs:137]: meat & potatoes

**Review by michaelglass — APPROVED** [review]
looks good. we need to set up travis-ci I think

**Comment by BrianHicks** [comment]
@allenap 👋 hey, I saw you started this in your account and forked it over to the NRI repo. Was your intent for this to be a general tool with a fork for NRI-specific thingies? If that's so, I think the core change here (the one @michaelglass  commented on) is probably going to be necessary for the general tool side of the equation, and I'd be happy to make an additional PR with just that and without the nix/niv shuffling.

**Comment by allenap** [comment]
@BrianHicks Hiya 👋  In the back of my mind I was trying to keep it as a general tool, but I've not done anything with it since July, and in any case it was first of all built for NoRedInk's environment. It makes sense that NoRedInk/firstaide should become the canonical repo, and I can deprecate allenap/firstaide.

**Comment by BrianHicks** [comment]
sounds like a plan, thank you! Hope everything's well for you. ❤️ 

FWIW, it clearly hasn't needed any modification prior to now—thank you for writing a reliable piece of software!

**Pull request merged**